.github/workflows/test-python.yml

```diff
@@ -63,7 +63,7 @@ jobs:
       - name: Start MongoDB
         uses: supercharge/mongodb-github-action@1.11.0
         with:
-          mongodb-version: 4.4
+          mongodb-version: 5.0
```

Comment:
Any reservation to running these tests using at least mongodb version 6.0? Or is the goal to test with our oldest supported version?

Reply:
4.4 was copied from the pymongo config. I bumped it here because `$dateTrunc` is new in 5.0. For this CI, I'd suggest using the oldest version we decide to support. That choice doesn't necessarily have to follow MongoDB's supported versions, but thus far, I haven't encountered anything that makes supporting 5.0 more difficult.